website/integrations/services/argocd/index.md

```diff
@@ -30,6 +30,8 @@ Only settings that have been modified from default have been listed.
 In authentik, create an _OAuth2/OpenID Provider_ (under _Applications/Providers_) with these settings:
 
 -   Name: ArgoCD
+-   Authorization flow: `default-provider-authorization-explicit-consent`
```

Comment:
we don't usually include the authorization flow in these docs as they aren't required to be set to a certain value for this to work

```suggestion
```

Reply:
Thanks.  Made the change and should be good now.